add_shutdown

Ver-ar opened the pull request.

Ver-ar committed:
add_shutdown

matemax commented on app/main.py:
```diff
@@ -23,6 +23,17 @@ async def launch_bot():
+async def cancel_me():
+    try:
+        app.state.polling_task.cancel()
+    except asyncio.CancelledError:
```
я что-то пропустил, апри каких условиях это может выполниться? кажется, что никогда
```python
import asyncio


async def f():
    await asyncio.sleep(1)
    print(1)

async def g():
    task = asyncio.create_task(f())
    await asyncio.sleep(0.5)
    try:
        res = task.cancel()
        print(res)
    except asyncio.CancelledError:
        print("exception") # не попадаем сюда никогда


asyncio.run(g())
```

Ver-ar replied:
это лишнее, удалила

Ver-ar committed:
Correct_shutdowm

-Исправила main, сейчас работает
-в models старые изменения, это я просто пыталась разобраться в алхимии, удалила пока

Ver-ar committed:
delete_unenecessary

-удалила комменты
-удалила ненужный код, который вытянулся со старой ветки, видимо

matemax commented on app/main.py:
```diff
@@ -20,36 +20,31 @@ async def launch_bot():
+            log.write ("Application shutdown")       
+
+logging.basicConfig(filename = 'log.log', format = '%(asctime)s-%(message)s', level=logging.DEBUG)
+logger = logging.getLogger()
```
посреди файла внезапно решили ...

matemax commented on app/database_process/models.py:
```diff
@@ -1,12 +1,12 @@
 #import asyncio
-from sqlalchemy.ext.asyncio import create_async_engine
-from sqlalchemy import Column, Integer, DateTime, MetaData, Sequence #, create_engine
+#from sqlalchemy.ext.asyncio import create_async_engine
```
и зачем это осталось?

Ver-ar committed:
correction_runtimeerror

Ver-ar committed:
delete_exception

Ver-ar committed:
add_asyncio.gather

Ver-ar committed:
Merge pull request #18 from Ver-ar/create_item_asyncio.gather

add_asyncio.gather
